xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp

```diff
@@ -813,25 +893,22 @@ void CGUIDialogPVRTimerSettings::DaysFiller(
   {
     list.clear();
     current = 0;
+    CDateTime time = pThis->m_minDateCovered;
 
-    // Data range: "today" until "yesterday next year"
-    const CDateTime now(CDateTime::GetCurrentDateTime());
-    CDateTime time(now.GetYear(), now.GetMonth(), now.GetDay(), 0, 0, 0);
-    const CDateTime yesterdayPlusOneYear(
-      time.GetYear() + 1, time.GetMonth(), time.GetDay() - 1, time.GetHour(), time.GetMinute(), time.GetSecond());
-
-    while (time <= yesterdayPlusOneYear)
+    list.push_back(std::make_pair("< " + time.GetAsLocalizedDate(), pThis->GetDateAsIndex(time - CDateTimeSpan(1, 0, 0, 0))));
```

Comment:
Now that solution (the concrete out of range value) I really like and isn't one I had considered.
I didn't know that would be possible using a 'generic' dates list - however if the range is specific to a particular timer (which it now is I believe) then this would work like a dream. Will do.